<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnakeDlg</class>
 <widget class="QDialog" name="SnakeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>LEFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>RIGHT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replayButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Replay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SnakeDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SnakeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
